pkg/scripts/render.go

```diff
@@ -184,79 +215,32 @@ var (
 			sudo yum-config-manager --save --setopt=docker-ce-stable.module_hotfixes=true
 			{{ end }}
 
-			sudo yum install -y yum-plugin-versionlock
```

Comment:
I don't think it's guaranteed that the `versionlock` plugin will be installed by default. We should still install it explicitly to be sure.

Reply:
It's removed here because it's the second time it's installed. So we still have versionlock plugin.